js/jquery.parallax.js

```diff
@@ -422,32 +423,38 @@
 		
 		return this.each(function(i) {
 			var node      = this,
-			    elem      = jQuery(this),
-			    opts      = args[i + 1] ? jQuery.extend({}, options, args[i + 1]) : options,
-			    decay     = opts.decay,
-			    size      = layerSize(elem, opts.width, opts.height),
-			    origin    = layerOrigin(opts.xorigin, opts.yorigin),
-			    px        = layerPx(opts.xparallax, opts.yparallax),
-			    parallax  = layerParallax(opts.xparallax, opts.yparallax, px),
-			    offset    = layerOffset(parallax, px, origin, size),
-			    position  = layerPosition(px, origin),
-			    pointer   = layerPointer(elem, parallax, px, offset, size),
-			    pointerFn = pointerOffTarget,
-			    targetFn  = targetInside,
-			    events = {
-			    	'mouseenter.parallax': function mouseenter(e) {
-			    		pointerFn = pointerOffTarget;
-			    		targetFn = targetInside;
-			    		timer.add(frame);
-			    		timer.start();
-			    	},
-			    	'mouseleave.parallax': function mouseleave(e) {
-			    		// Make the layer come to rest at it's limit with inertia
-			    		pointerFn = pointerOffTarget;
-			    		// Stop the timer when the the pointer hits target
-			    		targetFn = targetOutside;
-			    	}
-			    };
```

Comment:
NOTE: Fixing spacing issues here where both spaces and tabs were used.
 Add a line note